probe/main.go

```diff
@@ -80,16 +80,16 @@ func main() {
 	)
 
 	var (
-		weaveTagger  *tag.WeaveTagger
+		weaveTagger  *overlay.WeaveTagger
```

Comment:
Seems like a missed opportunity to have weaveTagger implement reporter, appended onto the reporters list and the special cases removed?